app.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"net/http"
+	"os"
+	"time"
+
+	fthealth "github.com/Financial-Times/go-fthealth/v1a"
+	oldhttphandlers "github.com/Financial-Times/http-handlers-go/httphandlers"
+	"github.com/Financial-Times/service-status-go/httphandlers"
+	"github.com/Sirupsen/logrus"
+	"github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
+	"github.com/jawher/mow.cli"
+)
+
+const serviceDescription = "A RESTful API which accepts Methode Articles and appends a vanity url before passing on to the UPP Stack"
+
+var timeout = time.Duration(10 * time.Second)
+var client = &http.Client{Timeout: timeout}
+
+func main() {
+	app := cli.App("methode-publish-handler", serviceDescription)
+	app.Version("methode-publish-handler", "0.0.0")
+
+	appName := app.StringOpt("app-name", "methode-publish-handler", "The name of this service")
+	appPort := app.StringOpt("app-port", "8084", "HTTP Port for the app")
+
+	notifierName := app.StringOpt("notifier", "cms-notifier", "The notifier service name")
+	notifierURL := app.StringOpt("notifier-url", "http://localhost:8080/notify", "The url for the notifier")
+	notifierPanicGuideURL := app.StringOpt("notifier-panic-guide", "", "The notifier panic guide url")
+	notifierHealthcheckURL := app.StringOpt("notifier-health-url", "http://localhost:8080/__health", "The notifier healthcheck url")
+
+	/*logMetrics := app.Bool(cli.BoolOpt{
+		Name:   "log-metrics",
+		Value:  false,
+		Desc:   "Whether to log metrics. Set to true if running locally and you want metrics output",
+		EnvVar: "LOG_METRICS",
+	})*/ // TODO: check if needed
+
+	app.Action = func() {
+		sc := ServiceConfig{
+			*appName,
+			*appPort,
+			*notifierName,
+			*notifierURL,
+			*notifierHealthcheckURL,
+			*notifierPanicGuideURL,
+		}
+		appLogger := NewAppLogger()
+		metricsHandler := NewMetrics()
+		contentHandler := NotifierHandler{&sc, appLogger, &metricsHandler}
+
+		handler := setupServiceHandler(sc, metricsHandler, contentHandler)
+
+		appLogger.ServiceStartedEvent(*appName, sc.asMap())
+		//metricsHandler.OutputMetricsIfRequired(*graphiteTCPAddress, *graphitePrefix, *logMetrics) // TODO: check if needed
+
+		err := http.ListenAndServe(":"+*appPort, handler)
+
+		if err != nil {
+			logrus.Fatalf("Unable to start server: %v", err)
+		}
+	}
+	app.Run(os.Args)
+}
+
+func setupServiceHandler(config ServiceConfig, metricsHandler Metrics, notifierHandler NotifierHandler) *mux.Router {
+	r := mux.NewRouter()
+
+	decoratedHandler := oldhttphandlers.HTTPMetricsHandler(metricsHandler.registry, oldhttphandlers.TransactionAwareRequestLoggingHandler(logrus.StandardLogger(), notifierHandler))
+	r.Path("/notify").Handler(handlers.MethodHandler{"POST": decoratedHandler})
```

Comment:
_I am wearing the heat of the REST nazi._
In theory, a RESTful resource should be a noun, not a verb, which means that `notifications` is more appropriated than `notify` in this case. 
But I also understand that `notifications` can be confusing because we usually use this term for notifications that we offer to our API consumers.
Does it make sense?